pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	defaultAPIVersion                      = "cloudwatch.aws.amazon.com/v1alpha1"
-	defaultInstrumenation                  = "java-instrumentation"
-	defaultNamespace                       = "default"
-	defaultKind                            = "Instrumentation"
+	defaultAPIVersion     = "cloudwatch.aws.amazon.com/v1alpha1"
+	defaultInstrumenation = "java-instrumentation"
```

Comment:
This is going to look very weird. A follow up to unravel the implications of changing this for existing customers would be good so we atleast understand.

Reply:
Actually we never actually create this resource on the cluster.. its just referenced in memory.
So it might be ok to rename this.

We need to test out the scenario to be sure... but this is non-blocking.

Reply:
you mean the naming of the instrumentation cr itself? yeah because this is no longer just java.

Reply:
yup thats right since this is no longer just java and currently we call it `java-instrumentation`